RELEASE.md

```diff
@@ -0,0 +1,18 @@
+Release type: minor
+
+Support for schema-extensions in subscriptions.
+
```

Comment:
We should mention that this is a breaking release, requiring all extensions to be instanciated and possibly all extension hooks to be updated.

We could possible use our favorite trick of making the execution_context argument requestable to stay backwards compatible. And when extensions try to access self.execution_context we could raise a deprecation warning.

Reply:
Actually I intentionally leaved it for another PR. the current extensions behavior remains unchained.
Do you think it should be done here?

Reply:
> the current extensions behavior remains unchained.

I'm sorry, I got tricked by GitHub's review UI. I thought the code instantiating extensions if needed was gone, but it has just been moved.